libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/api/ElasticsearchEntitlementManager.java

```diff
@@ -9,25 +9,49 @@
 
 package org.elasticsearch.entitlement.runtime.api;
 
-import org.elasticsearch.entitlement.api.EntitlementChecks;
-import org.elasticsearch.entitlement.api.EntitlementProvider;
+import org.elasticsearch.entitlement.bridge.EntitlementChecker;
+import org.elasticsearch.logging.LogManager;
+import org.elasticsearch.logging.Logger;
 
 import java.util.Optional;
 
 import static org.elasticsearch.entitlement.runtime.internals.EntitlementInternals.isActive;
 
 /**
- * Implementation of the {@link EntitlementChecks} interface, providing additional
+ * Implementation of the {@link EntitlementChecker} interface, providing additional
  * API methods for managing the checks.
  * The trampoline module loads this object via SPI.
  */
-public class ElasticsearchEntitlementManager implements EntitlementChecks {
+public class ElasticsearchEntitlementManager implements EntitlementChecker {
```

Comment:
The Elasticsearch in the name feels out of place now with all the other classes starting with Entitlement.

Reply:
My preference is to rename it, but I'd appreciate hearing @rjernst's opinion as well.

Reply:
Alright, if this works for you... I'll call it `ElasticsearchEntitlementChecker` and we can rename it again if need be once we hear Ryan's thoughts?

Reply:
I'd leave it as is - renaming can be done at any phase.

Reply:
Oh I did it already. 😅